src/_components/ForNPOs/ProjectCriteria.jsx

```diff
@@ -0,0 +1,126 @@
+import React from "https://esm.sh/react";
+
+export default function ProjectCriteria({ comp }) {
+  return (
+    <figure className="flex-col flex w-full lg:p-5 lg:max-w-[1280px]">
```

Comment:
You can delete the fixed size: `lg:max-w-[1280px]` and instead just add more padding: `lg:p-24`. This will ensure that the content remains centered even on larger screen sizes.

Reply:
I would add an additional breakpoint for xl and then one for lg so that the content is sized correctly

Reply:
For xl - xl:px-64